skietbaan_fe/src/components/CreateComp.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Jumbotron, Form, Button } from 'react-bootstrap';
+import propTypes from 'prop-types';
+import { connect } from 'react-redux';
+import '../scss/createcomp.css';
+import { createcomp } from '../actions/competition.action';
+class CreateComp extends Component {
+	constructor() {
+		super();
+		this.state = { compTitle: '', compStatus: '' };
+		//binding the onChange method to this commponents
+		this.onChange = this.onChange.bind(this);
+		this.onSubmit = this.onSubmit.bind(this);
+	}
+	//------------------------------------------------------------------------
+	/** A method that detects the change in the change in thw textfield */
+	onChange(e) {
+		this.setState({ [e.target.name]: e.target.value });
+	}
+	//------------------------------------------------------------------------
+	/** A method that handles the submit enent for the submit button*/
+	onSubmit(e) {
+		/** Preventing the default button action event to occur automatically*/
+		e.preventDefault();
+		const compData = {
+			compTitle: this.state.compTitle,
+			compStatus: true
```

Comment:
compStatus seems to be a boolean variable.
Set its initial value to false if you intend to toggle it

Reply:
(True) is a default value

Reply:
It has the same name as one of the state properties. I see.